content/content-strategy/identify-business-needs/getting-agreement/index.yml

```diff
@@ -0,0 +1,17 @@
+pagetitle: Getting agreement
+description: Abstract
```

Comment:
Missing metadata for `description`. Update prior to release.

Reply:
Also missing (possibly deliberate design choice): in `main` there is no table of contents file. Should there be? All so far have had one. I'm asking because there are multiple `nav/section` partials for the page and these can be accessed from a table of contents by users who want to skip directly via anchor link to the relevant section. I get the sense that these are shorter pages though?

Reply:
Fixed and checked.